.github/workflows/kindIntegTest.yml

```diff
@@ -126,18 +241,23 @@ jobs:
         version:
         - "3.11.14"
         - "4.0.7"
+        - "6.8.29"
         include:
           - version: 3.11.7
             serverImage: k8ssandra/cass-management-api:3.11.7-v0.1.24 # k8ssandra 1.1
           - version: 4.0.0
             serverImage: k8ssandra/cass-management-api:4.0.0-v0.1.28 # k8ssandra 1.3
+          - version: 6.8.29
+            serverImage: datastax/dse-mgmtapi-6_8:6.8.29-jdk8 # DSE 6.8.29
```

Comment:
Why is this DSE version running smoke_test_oss instead of smoke_test_dse? If smoke_test_dse is no longer necessary - remove it. If test_all_things_oss works for DSE also, then do we really need smoke_test_oss anymore? 

(some of these oddities can be addressed in the future tickets, but then those tickets must be created)

Reply:
I've removed `smoke_test_oss` and `smoke_test_dse`, and renamed `oss_test_all_the_things` to `test_all_the_things` as that test does pass against C* 3.11 and 4.0 as well as DSE 6.8. I put `test_all_the_things` as the test for the smoke tests now.